corehq/apps/hqadmin/templates/hqadmin/raw_couch.html

```diff
@@ -0,0 +1,34 @@
+{% load hq_shared_tags %}{% load compress %}
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Doc {{ doc_id }}</title>
+        {% compress css %}
+        <link type="text/less" rel="stylesheet" media="all" href="{% static 'style/less/hqstyle-core.less' %}"/>
+        {% endcompress %}
+        {% if less_debug %}
+            <script src="{% static 'style/lib/lessjs/less-1.3.1.min.js' %}"></script>
```

Comment:
yea, `style/static/style/lib/bootstrap-3.2.0/dist/css/bootstrap.min.css` haven't moved that over to bower yet